.github/workflows/lnp.yml

```diff
@@ -0,0 +1,31 @@
+name: L&P Tests
+
+on:
+  push:
+    branches:
+      - master 
+      - release
+jobs:
+  LnP-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Create Config Files
+        run: ./gradlew createConfigs
+      - name: Start App Engine
+        run: ./gradlew appengineStart
+      - name: Start Tests
+        run: ./gradlew lnpTests
```

Comment:
Currently this will not pass in GitHub Actions without also incorporating the fixes from fb8177cd9.

Reply:
Ah I see, okay updated based on the commit